src/amg/azext_amg/backup.py

```diff
@@ -130,57 +95,27 @@ def _save_dashboard_setting(dashboard_name, file_name, dashboard_settings, folde
     logger.info("    -> %s", file_path)
 
 
-def _get_individual_dashboard_setting_and_save(dashboards, folder_path, log_file, grafana_url, http_headers):
-    file_path = folder_path + '/' + log_file
-    if dashboards:
-        with open(file_path, 'w', encoding="utf8") as f:
-            for board in dashboards:
-                board_uri = "uid/" + board['uid']
-
-                (status, content) = get_dashboard(board_uri, grafana_url, http_headers)
-                if status == 200:
-                    # do not back up provisioned dashboards
-                    if content['meta']['provisioned']:
-                        logger.warning("Dashboard: \"%s\" is provisioned, skipping...", board['title'])
-                        continue
-                    _save_dashboard_setting(
-                        board['title'],
-                        board_uri,
-                        content,
-                        folder_path)
-                    f.write(board_uri + '\t' + board['title'] + '\n')
-
-
 # Save library panels
 def _save_library_panels(grafana_url, backup_dir, timestamp, http_headers, **kwargs):  # pylint: disable=unused-argument
     folder_path = f'{backup_dir}/library_panels/{timestamp}'
     log_file = f'library_panels_{timestamp}.txt'
+    log_file_path = folder_path + '/' + log_file
 
     if not os.path.exists(folder_path):
         os.makedirs(folder_path)
 
-    current_page = 1
-    while True:
-        panels = _get_all_library_panels_in_grafana(current_page, grafana_url, http_headers)
+    all_panels = get_all_library_panels(grafana_url, http_headers)
```

Comment:
do we include folders_to_include & folders_to_exclude arguments for the library panel save?

Reply:
Nope, backup never had that feature for library panels so I didn't add it. But in migrate, it is handled by using set()